catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfAttribute.java

```diff
@@ -36,6 +36,8 @@
    * @return a function that, given the CommonNitfSegment of type T, will return the corresponding
    *     value for the NitfAttribute as a Serializable. The function should return <code>null</code>
    *     if the value cannot be computed.
+   * @throws org.codice.alliance.transformer.nitf.NitfParsingException when there is an error
```

Comment:
✏️ If we include the exception in the signature (next comment), we shouldn't have to fully qualify the exception here anymore.